code/modules/surgery/organ_manipulation.dm

```diff
@@ -118,13 +99,6 @@
 	else if(implement_type in implements_extract)
 		current_type = "extract"
 		var/list/organs = target.getorganszone(target_zone)
-
-/*		var/mob/living/simple_animal/borer/B = target.has_brain_worms()
```

Comment:
Just a note that this is borers, but since borers aren't supported it can go and stay go.